native/dosbox-jsdos/jsdos/include/jsdos-timer.h

```diff
@@ -9,4 +9,6 @@ namespace jsdos {
   void initTimer();
 }
 
+void DelayWithYield(int);
```

Comment:
Do you need it? I don't see where it's called...

Reply:
You're right -- thank you.